### ISSUE TITLE
Password strength on wallet creation

### ISSUE THREAD

**Issue by flomang** [mimblewimble/grin-gui#39]
Wallet password creation needs requirements - min length, special characters, etc.

**Comment by 30-r** [comment]
I would like if there is an option to disable complexity requirements and even create no password wallets

**Comment by flomang** [comment]
Perhaps this is possible. Maybe someone else can chime in. How would a no password wallet be used?

**Comment by 30-r** [comment]
Yeah if no password is not easily added because grin-wallet doesn't already support it then no problem. I just always liked that Bitcoin Core did allow it, in a sort of "choose your own security" sense.
